Source/MLX/Documentation.docc/Articles/running-on-ios.md

```diff
@@ -70,3 +70,34 @@ Tune this value for your needs.
 
 Finally, if the device your code runs on has more RAM than the jetsam limit would
 normally allow, you can use the [Increased Memory Limit](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_kernel_increased-memory-limit) entitlement.
+
+## Developing for iOS
```

Comment:
Add suggestions from:

- https://github.com/ml-explore/mlx-swift-examples/issues/21

Should address 

- https://github.com/ml-explore/mlx-swift/issues/46
- 
as well.